### PR TITLE
Fixed bug to allow search to return more than 10 items

### PR DESCRIPTION
**Pull request by shengdi**
I have been using gplaycli to search and download large numbers of apps from the app store. But recently the search only returns a maximum of 10 items. This pull request makes some quick and shoddy changes to request for the next pages and combine the items together to return more matching applications based on "-n", this breaks API compatibility with googleplay_api though, but that code is not being maintained anymore.

### CODE REVIEW

**Comment by matlink** [comment]
Well, how did you implemented it? `-n` does not return the expected number of results.
```
$ gplaycli -n 20 -s firefox | wc -l
48
```

**Comment by shengdi** [comment]
Ah sorry, so basically I added a loop to request for the next page `message.payload.searchResponse.doc[0].containerMetadata.nextPageUrl`, but it seems like the number of applications returned actually varies from the first page to the next, added counting to make sure -n returns the right number.

Should be working properly now.

File: ext_libs/googleplay_api/googleplay.py [hunk 1]
```diff
@@ -194,13 +194,27 @@ def executeRequestApi2(self, path, datapost=None, post_content_type="application
...
         message = self.executeRequestApi2(path)
-        return message.payload.searchResponse
+
+        remaining = int(nb_results) - len(message.payload.searchResponse.doc[0].child)
+
+        messagenext = message
+        allmessages = message
+
+        while remaining > 0:
+            print(remaining)
```
Comment (matlink) [inline ext_libs/googleplay_api/googleplay.py]: don't need this print

File: ext_libs/googleplay_api/googleplay.py [hunk 2]
```diff
@@ -194,13 +194,27 @@ def executeRequestApi2(self, path, datapost=None, post_content_type="application
...
+            pathnext = messagenext.payload.searchResponse.doc[0].containerMetadata.nextPageUrl
+            messagenext = self.executeRequestApi2(pathnext)
+            remaining -= len(messagenext.payload.searchResponse.doc[0].child)
+            allmessages.MergeFrom(messagenext)
+          
+          
+
+
```
Comment (matlink) [inline ext_libs/googleplay_api/googleplay.py]: remove these blank lines

**Comment by matlink** [comment]
Hum, I'll revert it, because it sometimes throws errors: 
```
$ gplaycli -s "com.yogavpn"
Traceback (most recent call last):
[...]
    remaining -= len(messagenext.payload.searchResponse.doc[0].child)
IndexError: list index out of range

$ gplaycli -s "com.yogavpn" -n 15
Traceback (most recent call last):
[...]
    remaining -= len(messagenext.payload.searchResponse.doc[0].child)
IndexError: list index out of range
```

**Pull request merged**